### DISCUSSION TITLE
Content pasring behavior implementation

### DISCUSSION THREAD

**Discussion by mdhthahmd** [tailwindlabs/tailwindcss#10128] [Help] [answered]
hi,
i was reading the docs and came across, tailwind content parsing [behavior](https://tailwindcss.com/docs/content-configuration#class-detection-in-depth), and now i am curious how its implemented but couldn't find any references on the actual implementation from the repo, does anyone know how its implemented?

**Comment by wongjn** [comment, marked as answer]
https://github.com/tailwindlabs/tailwindcss/blob/41e655dd0b05396ead2b7923f25a6eb5e69680a2/src/lib/expandTailwindAtRules.js#L46-L74

**Reply by mdhthahmd** [reply]
thanks @wongjn
